src/main/java/org/hl7/davinci/priorauth/FhirUtils.java

```diff
@@ -442,4 +445,15 @@ private static boolean securityIsSubsetted(Bundle bundle) {
     }
     return false;
   }
+
+  /**
+   * @param bundle - the bundle resource
+   * @return if the request is actually a claim inquiry request
+   */
+  public static boolean isClaimInquiry(Bundle bundle) {
+    if (bundle.getMeta().getId().contains("inquiry"))
```

Comment:
I don't think this is a reliable way to know it is a claim inquiry. 
EDIT: I see below this is used to determine if the Claim to the $submit endpoint is an inquiry or not. The inquiry is an entirely new operation and that is how we know it is an inquiry (meaning this method can be deleted)